skinny/pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
   "cachetools<6,>=5.0.0",
   "click<9,>=7.0",
   "cloudpickle<4",
-  "databricks-sdk<1,>=0.20.0",
+  "databricks-sdk<1,>=0.41.0",
```

Comment:
FYI -- `databricks-sdk` on DBR/MLR can be quite old. by requiring this, we'll force a newer `databricks-sdk` version in all DBR/MLR that have newer MLflow. will that be an issue?

Reply:
Good question. It may cause an issue. For smaller files, older SDKs should work. Let me check.